pkgs/tools/archivers/7zz/default.nix

```diff
@@ -54,10 +54,9 @@ stdenv.mkDerivation rec {
   sourceRoot = ".";
 
   patches = [
-    ./fix-build-on-darwin.patch
     ./fix-cross-mingw-build.patch
+    ./fix-empty-expr-stmt.patch
```

Comment:
Could you please add some comments on the reason why this patch is needed ? It is a patch that we can upstream ? When should we remove it?

More info: https://github.com/jtojnar/nixpkgs-hammering/blob/main/explanations/missing-patch-comment.md

Reply:
Not sure how you ran `nixpkgs-hammering`, as it's broken with `finalAttrs` and known as such for over a year (jtojnar/nixpkgs-hammering#131). Anyway comment added

Commits have also been split as requested

Reply:
Actually, I don't use it. I just know the all the defined rules in the project and I like the explanations document... so I use it when I need to justify a feedbackk